src/dashboard/src/components/administration/views_processing.py

```diff
@@ -64,6 +72,57 @@ def write_to_file(self, file_path):
         file = open(file_path, 'w')
         file.write(etree.tostring(self.xml, pretty_print=True))
 
+
+def save__normalize(name, value, choice_uuid, do_not_lookup, xml_choices, default_processing_func):
+    """
+    There are two "Normalize" chain links with choices "Do not normalize", refs #10216.
+    """
+    if value != 'Do not normalize':
+        default_processing_func(value, choice_uuid, do_not_lookup, xml_choices)
+        return
+    sql = """
+        SELECT
+            MicroServiceChainLinks.pk,
+            MicroServiceChains.pk
+        FROM TasksConfigs
+        LEFT JOIN MicroServiceChainLinks ON (MicroServiceChainLinks.currentTask = TasksConfigs.pk)
+        LEFT JOIN MicroServiceChainChoice ON (MicroServiceChainChoice.choiceAvailableAtLink = MicroServiceChainLinks.pk)
+        LEFT JOIN MicroServiceChains ON (MicroServiceChains.pk = MicroServiceChainChoice.chainAvailable)
+        WHERE
+            TasksConfigs.description = 'Normalize'
+            AND MicroServiceChains.description = 'Do not normalize';
+    """
+    with connection.cursor() as cursor:
+        cursor.execute(sql)
+        for index, row in enumerate(cursor):
+            logger.debug('save__normalize(): appliesTo=%s goToChain=%s', row[0], row[1])
+            xml_choices.add_choice(row[0], row[1], comment='{} (match {} for "{}")'.format(name, index+1, value))
+
+
+def save_select_field(field, request, xml_choices):
+    try:
+        name = field['name']
+        value = request.POST.get(field['name'])
+        choice_uuid = field['choice_uuid']
+        do_not_lookup = field.get('do_not_lookup', False)
+    except KeyError:
+        logger.warning('save_select_field() is missing parameters')
+        return
+    # Default processing method for a select field
+    def default_processing(value, choice_uuid, do_not_lookup, xml_choices):
+        target = value if do_not_lookup else uuid_from_description(value, choice_uuid)
+        xml_choices.add_choice(choice_uuid, target, comment=name)
+    # Attempt to invoke custom callable if defined
+    save_callable_name = 'save__{}'.format(name)
+    save_callable = getattr(sys.modules[__name__], save_callable_name, None)
```

Comment:
This is fancy, but also feels fragile. Any new custom saves will have to be added by modifying this file anyway - why not make this an `if name == 'normalize': save_normalize(...)`  If we end up with lots of exceptions, we can look at something other than an if/else tree.  That would also removed the default_processing clojure, since it would just be the `else`.